README.rst

```diff
@@ -150,33 +150,70 @@ To update page objects installed from external repos:
 How to run Whitelabel tests
 ----------------------------
 
-Before running whitelabel tests please ensure that these environment variables are set.
+
+1. Before running the commands change your working directory to `edx-e2e-tests`.
+
+.. code:: bash
+
+    cd edx-e2e-tests/
+
+2. Install the requirements
+
+.. code:: bash
+
+    pip install -r requirements/base.txt
+
```

Comment:
Should add instructions to first create a virtualenv and do the installation from there